Api/Src/Services/MailingService.ts

```diff
@@ -0,0 +1,35 @@
+interface Message {
+    from: string,
+    to: string,
+    subject: string,
+    text: string,
+};
+
+interface Nodemailer {
+    createTransport({service, auth:{user, pass}})
```

Comment:
We could add @types/nodemailer library to our project for better typing, ofc it's just a suggestion.

Reply:
makes sense, I was trying to decouple the service from nodemailer itself but in the end I don't see any advantages of taking this approach - but as it's already done I'd keep it as it is, the current approach doesnt break anything and I don't need to redo the tests :)